src/vivarium/framework/results/manager.py

```diff
@@ -89,12 +89,11 @@ def on_post_setup(self, _: Event):
                     }
                     # Get the complete cartesian product of stratifications
                     index_values = list(itertools.product(*stratification_values.values()))
-                    self._metrics[measure] = pd.DataFrame(
-                        data=0.0,  # Initialize to 0
-                        columns=["value"],
+                    self._metrics[measure] = pd.Series(
```

Comment:
From discussions w/ Rajan: Let's go back to DataFrames b/c we probably want those later when we implement a different type of updater